torch/onnx/symbolic_opset9.py

```diff
@@ -103,11 +103,11 @@ def mul(g, self, other):
 
 
 def div(g, self, other):
-    return g.op("Div", self, other)
+    return true_divide(g, self, other)
```

Comment:
I am a bit confused as to whether these are needed in this change.

Reply:
yes, this is actually from the PR #44831. Included here to pass the scripting test with shape inference enabled for div. I'll mention #44831 as a dependency as well.